lib/Installation/AddOnProduct/AddOnProductController.pm

```diff
@@ -11,6 +11,7 @@ package Installation::AddOnProduct::AddOnProductController;
 use strict;
 use warnings;
 use Installation::AddOnProduct::AddOnProductPage;
+use Installation::AddOnProduct::AdditionalProductPage;
```

Comment:
please, change to 2022 the copyright year in all the files.

Reply:
Changed